### ISSUE TITLE
Where can I get the images under the corresponding path of EMBED_IMG_PATH in config.py?

### ISSUE THREAD

**Issue by silenceyese** [kvsnoufal/Pytorch-FaceNet-DogDataset#1]
Hello, thank you very much for this project. I tried to train the model without EMBED_IMG_PATH for the images. The log is as follows:

TRAIN_DATASET : 6123 VALIDATION_DATASET : 2240 EMBEDDING_LOG_DATASET: 0

Final training results:

Training STEP: 0 LOSS: 0.9843002078199231
Training STEP: 0 LOSS: 1.017400431313685
Saving Best Model  0

This should be caused by the missing of EMBEDDING_LOG_DATASET pictures. I really hope to get your help. Thank you very much!
